app/api/auth.py

```diff
@@ -269,3 +273,20 @@ def change_password():
         "name": user.fullname if user.fullname else None,
         "password-changed": True
     })
+
+
+def authorized_access_ticket(ticket_function):
+    @wraps(ticket_function)
+    def wrapper(*args, **kwargs):
+        if current_identity.is_anonymous and current_user.id == Order.query.filter_by(identifier=kwargs['identifier']).first().user.id:
```

Comment:
Wrong logic?

Reply:
how do we check if thhe current user is associated with the ticket?

Reply:
Here you'll throw an error, by your own logic, 

1. If there is no logged in user, AND
2. Current user's ID matches with the one who placed the order

Seems something wrong with this logic?

Reply:
Just a quick doubt, Is there a possibility that if there is no logged in user, the current_user can still be present in the session?

Reply:
No, that's why this does not make any sense